lib/ChunkTemplate.js

```diff
@@ -12,6 +12,8 @@ function ChunkTemplate(outputOptions) {
 module.exports = ChunkTemplate;
 
 ChunkTemplate.prototype = Object.create(Template.prototype);
+ChunkTemplate.prototype.constructor = ChunkTemplate;
```

Comment:
> Like i said i might be totally wrong, correct me if so.

You are wrong. The code is fine. I doesn't call the constructor. It just sets the property `constructor` to the constructor function.